lib/src/pages/login_page.dart

```diff
@@ -0,0 +1,122 @@
+
+
+// ignore_for_file: prefer_const_constructors
+
+import 'package:flutter/material.dart';
+import 'package:flutter_bloc/flutter_bloc.dart';
+import 'package:fluttertoast/fluttertoast.dart';
+import 'package:loginform_new/bloc/auth_cubit.dart';
+import 'package:loginform_new/src/pages/dashboard_page.dart';
+
+class HomePage extends StatefulWidget {
+  const HomePage({Key? key}) : super(key: key);
+
+  @override
+  _HomePageState createState() => _HomePageState();
+}
+
+class _HomePageState extends State<HomePage> {
+  TextEditingController emailController = TextEditingController();
+  TextEditingController passwordController = TextEditingController();
+  GlobalKey<FormState> formKey = GlobalKey();
+
+  bool obscureText = true;
+  @override
+  Widget build(BuildContext context) {
+    return Scaffold(
+      body: ListView(children: [
+        Padding(
+          padding: const EdgeInsets.all(8.0),
+          child: SafeArea(
+            child: Form(
+                key: formKey,
+                child: Column(
+                  children: [
+                    SizedBox(height: 8),
+                    TextFormField(
+                      validator: (val) {
+                        String data = val ?? '';
+                        if (data.trim().isEmpty) {
+                          return ' Please enter email';
```

Comment:
Find a regular expression for email validation. https://stackoverflow.com/questions/16800540/validate-email-address-in-dart